docs/installation_configuration_guide/DeploymentManagerConfiguration.md

```diff
@@ -282,7 +281,7 @@ Deployment Manager of type `lite-embedded` has the following configuration optio
 
 | Parameter                                                 | Type   | Default value   | Description                                                                                                                                              |
 |-----------------------------------------------------------|--------|-----------------|----------------------------------------------------------------------------------------------------------------------------------------------------------|
-| mode                                                      | string |                 | Either streaming or request-response                                                                                                                     |
+| mode                                                      | string |                 | (Processing) mode: either  streaming-lite or request-response                                                                                                                     |
```

Comment:
1. The same comment about parenthesis
2. WDYT about some refactoring here? We have 4 columns: parameter, type, default value, description. Some params are rather enums (but based on string type). It means that it'd be nice to have a separate column where we could define the "possible values" or sth. To not add them to the description column

Reply:
removed parentheses